src/test/resources/regressions/features/ghost_pointer/ghost-reference-simple01.gobra

```diff
@@ -0,0 +1,31 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package GhostReferenceSimple01
+
+type Test struct {
+    field int
+}
+
+ghost
+decreases
+ensures acc(x)
+func GhostFunc() (x gpointer[Test]) {
+    x = &Test{ 42 }
```

Comment:
Do we support `x = nil`? If yes, then add a test for that. If no, then we might want to add this later (can be done in a later PR).

Reply:
yes we do. I'm adding a testcase